tools/js-sdk-release-tools/src/test/pipeline/sdkGeneration.test.ts

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import { describe, expect, test } from 'vitest';
+import { getRandomInt } from '../utils/utils';
+import { ensureDir, remove, writeFile } from 'fs-extra';
+import { stringify } from 'yaml';
+import { SDKType } from '../../common/types';
+import { parseInputJson } from '../../utils/generateInputUtils';
+
+describe('Auto SDK generation path test', () => {
+    test('high level client generation', async () => {
```

Comment:
I wonder if we could change the indent as two spaces? maybe not in this PR.

Reply:
Sure, I'll format the whole js-sdk-release-tool project in another pr.